.github/workflows/code_scan.yml

```diff
@@ -27,29 +27,25 @@ jobs:
           path: .tox/bandit-report.txt
         # Use always() to always run this step to publish scan results when there are test failures
         if: ${{ always() }}
-  Snyk-scan:
-    runs-on: [self-hosted, linux, x64]
-    permissions:
-      security-events: write
+  Trivy-scan:
+    runs-on: ubuntu-20.04
```

Comment:
if we want to upload scanning results to gh as the `sarif` format, we need to have `security-events` write permission so that could be added later again.

Reply:
Alright. Thanks for the clarification.